### PR TITLE
Inclusiveness, Diversity and Accessibility

### PR DESCRIPTION
**Pull request by maartenba**
Crew, please review and provide any comments you may have. We have a lot of things we have been doing, but let's put them in the open and clearly communicate what we can and can not do.

### CODE REVIEW

File: _pages/inclusiveness-diversity-accessibility.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+At AZUG, we care deeply about making our events a safe and welcoming space for everyone involved.
+
+Given AZUG is driven by volunteers, this does involve making trade-offs due to available time and effort. We want to prioritize those efforts that have the most impact on safety, comfort, diversity, and accessibility.
+
+On this page, we will list the things we are doing, those that we consider and those that we decided not to do for the time being.
+
+Do provide us your feedback on these - if you are affected by these efforts, if you have relevant experiences, or if you are simply a bystander who cares, let us know what you would like to see us do differently.
+
+Please send your ideas or concerns to [getintouchwithus@azug.be](getintouchwithus@azug.be). We treat all submissions confidentially.
```
Comment (t2kx) [inline _pages/inclusiveness-diversity-accessibility.md]: we treat all submissions confidentially and equally.

File: _pages/inclusiveness-diversity-accessibility.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+### Code of Conduct
+
+We ask all participants, speakers, sponsors and volunteers to adhere to [our Code of Conduct](https://www.azug.be/code-of-conduct) at all of our events. We also invite everyone to report any violations as soon as possible, so we can take appropriate action.
+
+### Contact Person
+
+Refer [our Code of Conduct](https://www.azug.be/code-of-conduct) for contact details regarding violations or questions.
+For general enquiries, ideas or concerns, e-mail us at [getintouchwithus@azug.be](getintouchwithus@azug.be).
+
+We treat all submissions confidentially.
```
Comment (t2kx) [inline _pages/inclusiveness-diversity-accessibility.md]: we treat all submissions confidentially and equally.

File: _pages/inclusiveness-diversity-accessibility.md (hunk 1 above)
Comment (maartenba) [reply _pages/inclusiveness-diversity-accessibility.md]: Thanks, fixed.

File: _pages/inclusiveness-diversity-accessibility.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+### Discounts or scholarships for students or underrepresented groups
+
+At the moment we have no scholarship program. We feel that given the topics, user group events and our conference mostly benefits people who work in the industry and can therefore afford to register.
+
+We are open to consider individual cases, so do get in touch via e-mail ([getintouchwithus@azug.be](getintouchwithus@azug.be)) if you feel we should reconsider.
+
+### Custom pronouns
+
+For every edition we reconsider if we should put custom pronouns on badges, but so far haven't found a compelling reason to do so.
```
Comment (KrisvanderMast) [inline _pages/inclusiveness-diversity-accessibility.md]: What's custom pronouns?

Comment (maartenba) [reply _pages/inclusiveness-diversity-accessibility.md]: More reading at https://blog.hubspot.com/marketing/gender-neutral-pronouns

**Pull request merged**